footer.php

```diff
@@ -10,8 +10,12 @@
  */
 
 $timber_context = $GLOBALS['timberContext'];
-if ( ! isset( $timber_context ) ) {
-	throw new \Exception( 'Timber context not set in footer.' );
+try {
```

Comment:
Do we use that file anywhere? @angtheod any thoughts?

Reply:
I didn't found this file used somewhere.

Reply:
Can we remove this and the page-plugin twig template?
@angtheod do you recall, this comes from timber starter theme?

Reply:
@kirdia Yes, page-plugin.twig comes from Timber. We are not really using those files `header.php footer.php page-plugin.twig` since we are using Timber.

Reply:
As we are not using `header.php footer.php page-plugin.twig` files, I am removing them.